tests/integration/test.basics.js

```diff
@@ -73,6 +73,20 @@ adapters.forEach(function (adapter) {
       return new PouchDB(dbs.name).destroy({});
     });
 
+    it('4339 throw useful error if method called on stale instance', function () {
+      var db = new PouchDB(dbs.name);
+
+      return db.put({
+        _id: 'cleanTest'
+      }).then(function () {
+        return db.destroy();
+      }).then(function () {
+        return db.get('cleanTest');
+      }).catch(function (err) {
+        should.equal(err instanceof Error, true, 'should be an error');
+      });
```

Comment:
It's a subtle difference, but this really ought to be

```js
.then(function () {
  throw new Error('expected an error');
}, function (err) {
  should.equal(err instanceof Error, true, 'should be an error');
});
```

Reason being: if the `db.get()` succeeds somehow, then the `catch()` function will never even run. It's not a big deal, though, and I can change it on the way in if you don't mind.